javascript/javascript2/week1/script.js

```diff
@@ -0,0 +1,102 @@
+// Exercise 1 - FIND THE SHORTEST WORD
+
+const danishWords = ["bil", "plante", "kaffe", "bog", "ø", "planetarium"];
+
+let shortestWord = "";
+
+function getShortestWord(array) {
+  for (let i = 0; i < array.length; i++) {
+    shortestWord = array[0];
+    if (array[i].length < shortestWord.length) {
+      shortestWord = array[i];
+      return shortestWord;
+    }
+  }
+}
+
+console.log(getShortestWord(danishWords));
+
+// FIND AND COUNT THE DANISH LETTERS
+const danishString = "Jeg har en blå bil";
+const danishString2 = "Blå grød med rød bær";
+const danishString3 = "";
+
+function findDanishLetters(sentence) {
+  const danishLetters = {};
+  const lettersArray = sentence.split("");
+
+  let countÆ = 0;
+  let countØ = 0;
+  let countÅ = 0;
+
+  for (let i = 0; i < lettersArray.length; i++) {
+    if (lettersArray[i] === "æ" || lettersArray[i] === "Æ") {
```

Comment:
What you could do in this check is instead of checking for both uppercase and lowercase you could take the item and use toUpperCase or toLowerCase and then you would not need to do a check for both uppercase and lowercase as the letter would have one format.